yarn.lock

```diff
@@ -1833,6 +1833,23 @@ __metadata:
   languageName: unknown
   linkType: soft
 
+"@automattic/site-admin@workspace:packages/site-admin":
+  version: 0.0.0-use.local
```

Comment:
Notice how the beta version is irrelevant unless we want to publish it and use it externally immediately.